verifier/src/report_body.rs

```diff
@@ -0,0 +1,177 @@
+// Copyright (c) 2018-2023 The MobileCoin Foundation
+
+//! Verifiers which operate on the [`ReportBody`](::mc_attest_core::ReportBody)
+//! data structure.
+
+use crate::{Result, VerificationStep};
```

Comment:
I had named the common trait `VerificationStep`.  Thinking about it, the "step" adjective feels like it's what the higher level container will see these as, not what they see themselves as. Also looking at the current Epid logic that uses `Verifier` I'm thinking this should  get renamed to `Verifier`.

Naming is hard and I'm up for other guidance

Reply:
This was fixed in #2